hyperstart/libhyperstart/persistence.go

```diff
@@ -0,0 +1,72 @@
+package libhyperstart
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+)
+
+type persistProcessState struct {
+	StdioSeq  uint64
+	StderrSeq uint64
+	StreamSeq uint64
+}
+
+type persistSerialBasedHyperstart struct {
+	LastStreamSeq uint64
+	Procs         map[string]persistProcessState
+}
+
+func (h *serialBasedHyperstart) Dump() ([]byte, error) {
+	h.RLock()
+	defer h.RUnlock()
+	p := &persistSerialBasedHyperstart{
+		LastStreamSeq: h.lastStreamSeq,
+		Procs:         make(map[string]persistProcessState, len(h.procs)),
+	}
+	for k, v := range h.procs {
+		p.Procs[fmt.Sprintf("%s@%s", k.c, k.p)] = persistProcessState{
+			StdioSeq:  v.stdioSeq,
+			StderrSeq: v.stderrSeq,
+			StreamSeq: v.stdinPipe.streamSeq,
+		}
+	}
```

Comment:
vmContext has all the info(xxxxSeq) of the containers, please use the info from vmContext.

and 'Dump()' is unneeded, libhyperstart has already provide LastStreamSeq(), please use that it.